snapstudysensei/windows_list/windows_list_win32.py

```diff
@@ -0,0 +1,31 @@
+import ctypes
+from ctypes import wintypes
+
+user32 = ctypes.windll.user32
+
+
+class WindowsList:
+    def __init__(self):
+        self.windows = {}
+
+    def __call__(self) -> dict[int, tuple[str, bool]]:
+        WNDENUMPROC = ctypes.WINFUNCTYPE(
+            wintypes.BOOL,
+            wintypes.HWND,
+            wintypes.LPARAM
+        )
```

Comment:
Can you make this a single line? (running black changes only that line; I need to add a CI isort/black linter workflow)